arelle/PackageManager.py

```diff
@@ -72,10 +93,7 @@ def parsePackage(cntlr, filesource, metadataFile, fileBase, errors=[]):
     _file = filesource.file(metadataFile)[0] # URL in zip, plain file in file system or web
     parser = lxmlResolvingParser(cntlr)
     try:
-        tree = etree.parse(_file,parser=parser)
-        # schema validate tp xml
-        xsdTree = etree.parse(TP_XSD,parser=parser)
-        etree.XMLSchema(xsdTree).assertValid(tree)
+        tree = _parseFile(cntlr, parser, metadataFile, _file, TP_XSD)
```

Comment:
What happens if you have taxonomy-package.xsd in the cache but are missing xml.xsd (`schemaRef`)?

Reply:
xml.xsd isn't loaded into the cache, but validation still completes successfully.

Reply:
Thanks.  I knew xml.xsd was optionalish, but I wasn't sure at what level it would be ignored.

Reply:
From the command line I get:

> [Exception] Entry point loading Failed to complete request:
Element '{http://www.w3.org/2001/XMLSchema}import': Failed to parse the XML resource 'http://www.w3.org/2001/xml.xsd'., line 16

Reply:
From the command line I get:

> [Exception] Entry point loading Failed to complete request:
Element '{http://www.w3.org/2001/XMLSchema}import': Failed to parse the XML resource 'http://www.w3.org/2001/xml.xsd'., line 16

Reply:
Discussed offline. The error I ran into doesn't appear to be caused by the package manager.